docs/faq.md

```diff
@@ -22,6 +23,70 @@ If a question you have is not answered below, please [submit an issue](/../../is
 
 See [rules_docker](https://github.com/bazelbuild/rules_docker) for a similar existing container image build tool for the [Bazel build system](https://github.com/bazelbuild/bazel). The tool can build images for languages such as Python, NodeJS, Java, Scala, Groovy, C, Go, Rust, and D.
 
+### How do I run the image I built?
+
+If you built your image directly to the Docker daemon using `jib:dockerBuild` (Maven) or `jibDockerBuild` (Gradle), you simply need to use `docker run <image name>`.
+
+If you built your image to a registry using `jib:build` (Maven) or `jib` (Gradle), you will need to pull the image using `docker pull <image name>` before using `docker run`.
+
+To run your image on Kubernetes, see [steps 4-6 of the Kubernetes Engine deployment tutorial](https://cloud.google.com/kubernetes-engine/docs/tutorials/hello-app#step_4_create_a_container_cluster).
+
+### How do I set parameters for my image at runtime?
+
+#### JVM Flags
+
```

Comment:
I think it's necessary to say this applies for `distroless/java`.  Other JRE images may require using other environment variables.

Reply:
Interesting. I thought this variable is natively supported (`https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/envvars002.html`), but not really?

Reply:
Not that I've ever used or tested it.

Reply:
One of the links in there leads here: https://docs.oracle.com/javase/8/docs/platform/jvmti/jvmti.html#tooloptions

which says "Platforms which support environment variables or other named strings, may support the JAVA_TOOL_OPTIONS variable." So I'm not sure it's guaranteed to work.

Reply:
https://stackoverflow.com/questions/28327620/difference-between-java-options-java-tool-options-and-java-opts
  - `_JAVA_OPTIONS` trumps `JAVA_TOOL_OPTIONS`, though it's OpenJDK/Oracle specific
  - IBM J9 and OpenJ9 support `IBM_JAVA_OPTIONS`
  - there's also a `JAVA_OPTS` that isn't supported directly by the JVM but is used by a lot of start scripts.